### ISSUE TITLE
Replace python3-dbus

### ISSUE THREAD

**Issue by qarkai** [OpenPrinting/system-config-printer#249]
According to https://www.freedesktop.org/wiki/Software/DBusBindings/ dbus-python is obsolete, recommended libraries are pydbus, txdbus or GDBus/QtDBus bindings. Since project uses GTK/GLib it seems that most appropriate replacement is GDBus.

**Comment by zdohnal** [comment]
Hi @qarkai ,

thanks for letting me know.

As for the migration I'm keeping system-config-printer in mainly maintenance mode and I'm not planning to invest work into migration - however, I'm open to pull request implementing the migration or give permissions to commit to a person willing to further develop system-config-printer.

I'm leaving this opened for the future.

**Comment by tinywrkb** [comment]
> I'm keeping system-config-printer in mainly maintenance mode

Is this related to CUPS 3.0 changes, meaning moving filters and driver into printer applications?  

Is it expected that with IPP Everywhere and printer applications, most users will not bother to configure CUPS, just run it locally (AF_UNIX support is in the 3.0 milestone), and only enterprise admins or advanced users will need to configure the daemon? So no need for a local GUI app for CUPS as the web UI is more suited for these users.

Can you post something more official, like adding some details to the README?

I thought about packaging this as a Flatpak, but if it's expected to be decommissioned, then I guess I shouldn't.

**Comment by zdohnal** [comment]
@tinywrkb 

it is connected to the changes in the matter of speaking - there still is an usage for local GUI app with CUPS 3.0, however it will require updates for s-c-p and pycups (basically s-c-p could start to show IPP services instead of queues if IPP services support is implemented, and pycups would need to use the current CUPS API), which I'm not able (and not willing due my interest in other projects) to deliver.

To sum it up, the project(s) can be used with newer CUPS, but it will need a person willing to dedicate his work on making it work with new CUPS - new developer or co-maintainer. Otherwise I'll let s-c-p in maintenance, willing to accept patches, which I'll test if they work and they don't break something.

**Comment by tinywrkb** [comment]
@zdohnal thanks for the detailed answer. Maybe it is worth posting this in a new sticky issue.

**Comment by zdohnal** [comment]
@tinywrkb I've updated README.md file - hopefully it covers what's the situation and what is needed.
